<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkReport</class>
 <widget class="QWidget" name="WorkReport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>710</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>985</width>
    <height>710</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Work Report</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 108);
</string>
  </property>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>650</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton#close:hover{
	background-color: rgb(209, 69, 0);
};
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 255, 0);</string>
   </property>
   <property name="text">
    <string>work report</string>
   </property>
  </widget>
  <widget class="QLabel" name="lineOne">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>891</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>lineOne</string>
   </property>
  </widget>
  <widget class="QLabel" name="lineTwo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>901</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>lineTwo</string>
   </property>
  </widget>
  <widget class="QLabel" name="lineThree">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>901</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>lineThree</string>
   </property>
  </widget>
  <widget class="QLabel" name="linefour">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>901</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>lineFour</string>
   </property>
  </widget>
  <widget class="QLabel" name="lineFive">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>901</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>linefive</string>
   </property>
  </widget>
  <widget class="QLabel" name="lineSix">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>901</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>lineSix</string>
   </property>
  </widget>
  <widget class="QLabel" name="lineSeven">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>901</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>lineSeven</string>
   </property>
  </widget>
  <widget class="QLabel" name="lineEight">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>901</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>lineEight</string>
   </property>
  </widget>
  <widget class="QLabel" name="lineNine">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>410</y>
     <width>901</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>lineNine</string>
   </property>
  </widget>
  <widget class="QLabel" name="lineTen">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>450</y>
     <width>901</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>lineTen</string>
   </property>
  </widget>
  <widget class="QLabel" name="lineEleven">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>490</y>
     <width>901</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>lineEleven</string>
   </property>
  </widget>
  <widget class="QLabel" name="lineTwoEleven">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>530</y>
     <width>901</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>lineTwoEleven</string>
   </property>
  </widget>
  <widget class="QLabel" name="line13">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>570</y>
     <width>901</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>line13</string>
   </property>
  </widget>
  <widget class="QLabel" name="line14">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>610</y>
     <width>901</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>line14</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
